reporter.go

```diff
@@ -116,9 +98,14 @@ func getAndValidateLambdaEnvironment() bool {
 		log.Panicf("Environment variable WAVEFRONT_API_TOKEN is not set.")
 	}
 
+	reportEnabled := os.Getenv("WAVEFRONT_ENABLED")
+	if strings.EqualFold(reportEnabled, "false") {
```

Comment:
if WAVEFRONT_ENABLED = false, ideally we don't need the otherwise mandatory env variables i.e. WAVEFRONT_URL / WAVEFRONT_API_TOKEN to be validated above. So we can move this validation to the top. If false, skip everything else ?

Reply:
I hadn't considered that, but you're absolutely right